src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -183,3 +175,11 @@ def print_revert_report(self, *, delete_managed: bool) -> bool | None:
             print("Migrated Manged Tables (targets) will be left intact.")
             print("To revert and delete Migrated Tables, add --delete_managed true flag to the command.")
         return True
+
+    def _get_mapping_rules_dict(self):
+        mapping_rules: dict[str, Rule] = {}
+        for rule in self._tm.load():
+            mapping_rules[
+                rule.as_hms_table_key
+            ] = rule
```

Comment:
```suggestion
            if self.is_upgraded(rule.src_schema, rule.src_table):
              continue
            if <<<<<read TableMapping.UCX_SKIP_PROPERTY and skip it>>>>>:
              continue
            mapping_rules[rule.as_hms_table_key] = rule
```

though, it should all be in the `TableMapping` class, because skip is handled there.

Reply:
Next PR